package-lock.json

```diff
@@ -1,78 +1,250 @@
 {
   "name": "lexical-models",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
   "requires": true,
   "packages": {
     "": {
       "name": "lexical-models",
       "version": "1.0.0",
       "license": "MIT",
       "dependencies": {
-        "@keymanapp/lexical-model-compiler": "^16.0.138",
-        "@keymanapp/models-types": "^16.0.138",
+        "@keymanapp/kmc": "17.0.192-alpha",
+        "@keymanapp/models-types": "17.0.192-alpha",
```

Comment:
My package-lock.json got updated to 17.0.222-alpha (numerous lines)

Reply:
yep, `npm install` once more sorts that. First time did not update the dependency versions, is it a bug in npm? No idea...